package.json

```diff
@@ -19,8 +22,10 @@
   },
   "homepage": "https://github.com/LambdaSchool/Sprint-Challenge-Node-Express#readme",
   "dependencies": {
+    "express": "^4.16.4",
     "helmet": "^3.16.0",
     "knex": "^0.16.5",
+    "morgan": "^1.9.1",
     "sqlite3": "^4.0.6"
   },
   "devDependencies": {
```

Comment:
remember to add in teh dev dependancy for nodemon for future devs who may not have it globally installed